src/components/spaces/views.test.tsx

```diff
@@ -382,6 +378,44 @@ describe(SpacesPage, () => {
     expect(queryByText(`There is 1 space in ${organization.entity.name}.`)).toBeTruthy();
   });
 
+  it('should ONLY show qouta usage calculation for org managers', () => {
+    const { container } = render(
+      <SpacesPage
+        linkTo={route => `__LINKS_TO__${route}`}
+        isAdmin={false}
+        isManager={true}
+        isBillingManager={false}
+        organization={organization}
+        spaces={[space, { ...space, quota: undefined }]}
+        users={[null]}
+      />,
+    );
+    expect(container.querySelectorAll('.org-summary-stat__value')[0]).toHaveTextContent('default');
+    expect(container.querySelectorAll('.org-summary-stat__value')[1]).toHaveTextContent('20.0%');
+    expect(container.querySelector('p')).toHaveTextContent(
+      'Using 1.00 GiB of memory out of a maximum of 5.00 GiB.',
+    );
+  });
+
+  it('should NOT show qouta usage calculation for NON org managers and only show assigned quota name', () => {
+    const { container } = render(
+      <SpacesPage
+        linkTo={route => `__LINKS_TO__${route}`}
+        isAdmin={false}
+        isManager={false}
+        isBillingManager={false}
+        organization={organization}
+        spaces={[space, { ...space, quota: undefined }]}
+        users={[null]}
+      />,
+    );
+    expect(container.querySelectorAll('.org-summary-stat__value')[0]).toHaveTextContent('default');
+    expect(container.querySelectorAll('.org-summary-stat__value')[0]).not.toHaveTextContent('20.0%');
+    expect(container.querySelector('p')).not.toHaveTextContent(
+      'Using 1.00 GiB of memory out of a maximum of 5.00 GiB.',
```

Comment:
Sometimes when negating assertions it's useful to be _less_ exact/specific with the condition. This for example would "pass" if the string were present but just missing the final full stop.

Reply:
good point. I've update the test to check for presence of that container